Installomator.sh

```diff
@@ -635,9 +635,14 @@ case $label in
         updateTool="/Library/Application Support/Microsoft/MAU2.0/Microsoft AutoUpdate.app/Contents/MacOS/msupdate"
         updateToolArguments=( --install --apps WDAV00 )
         ;;
+    powershell)
+        name="PowerShell"
+        type="pkg"
+        downloadURL=$(downloadURLFromGit PowerShell PowerShell | grep lts)
```

Comment:
The PowerShell project has 2 packages for macOS.
I chose lts package because I thought lts was more appropriate.